src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleControl.cs

```diff
@@ -56,12 +56,16 @@ private static void OnSampleContentChanged(object dependencyObject, DependencyPr
 		}
 
 #if XAMARIN
-		protected override void OnDataContextChanged()
+		internal override bool GetDefaultValue2(DependencyProperty property, out object defaultValue)
```

Comment:
This should be a breaking change, I'm surprised the validation does not catch it. Let's keep the `OnDataContextChanged` method and mark for next major.

Reply:
@jeromelaban This is SamplesApp, so should be okay